lib/rubocop/cop/rspec/empty_hook.rb

```diff
@@ -27,12 +27,14 @@ class EmptyHook < Cop
 
         MSG = 'Empty hook detected.'
 
-        def_node_matcher :empty_hook?, <<~PATTERN
-          (block $#{Hooks::ALL.send_pattern} _ nil?)
+        def_node_matcher :empty_block?, <<~PATTERN
```

Comment:
Do we really need a matcher to tell that a block is empty?
`add_offense(hook) is node.body.nil?` should do the job